Enterprise-CMCS/eAPD
# 4597 - [Feature] Display standard APD header on activities pages

Resolves #4597 

### Description
The APD header is not being shown on the activities pages, but it should be shown on those pages as well.

### Chromatic Link
https://www.chromatic.com/build?appId=61d5b948cf6f17003a12bf77&number=1359

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| Continue button test | running in e2e | will check for header on every page | cypress |

### Steps to manually verify this change

1. Open a new or existing apd
2. Go to an activity page
3. Ensure apd name header is present.

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
